src/main/java/org/generationcp/middleware/service/impl/study/StudyInstanceServiceImpl.java

```diff
@@ -217,6 +221,10 @@ public InstanceObservationData addInstanceObservation(final InstanceObservationD
 			new Phenotype(variableId, instanceObservationData.getValue(), experimentModel);
 		phenotype.setCreatedDate(new Date());
 		phenotype.setUpdatedDate(new Date());
+
```

Comment:
Also consider the phenotypes created from SSA and Dataset Importer in updating the updated/created by fields.